enterprise/ha/src/main/java/org/neo4j/kernel/impl/ha/ClusterManager.java

```diff
@@ -102,6 +104,13 @@
 public class ClusterManager
         extends LifecycleAdapter
 {
+    /**
+     * _Network _Flags for passing into {@link ManagedCluster#fail(HighlyAvailableGraphDatabase, int)}
+     * NF_OUT: Fail outgoing cluster network traffic.
+     * NF_IN: Fail incoming cluster network traffic.
+     */
+    public static final int NF_OUT = 0x1, NF_IN = 0x2;
```

Comment:
can we please use enum instead of flags? that gives more clear feeling what method is expected compare to int.